jina/importer.py

```diff
@@ -95,6 +96,57 @@ def __exit__(self, exc_type, exc_val, traceback):
                 return True  # suppress the error
 
 
+class ExtensionFileLoader(Loader):
+    """Loader for extension modules."""
+
+    @classmethod
+    def exec_module(cls, module):
+        """Initialize an extension module
+
+        :param module: a namespace module
+        """
+        load_path = module.__spec__.origin
+        init_file = '__init__.py'
+        if load_path.endswith(init_file):
+            module.__path__ = [load_path[: -len(init_file) - 1]]
```

Comment:
For my understanding, what's the use case for setting the `module.__path__` variable?